src/Estimators/EstimatorManagerInput.h

```diff
@@ -58,8 +59,13 @@ using ScalarEstimatorInputs = std::vector<ScalarEstimatorInput>;
 class EstimatorManagerInput
 {
 public:
-  EstimatorManagerInput()                            = default;
-  EstimatorManagerInput(EstimatorManagerInput&& emi) = default;
+  EstimatorManagerInput()                                 = default;
+  EstimatorManagerInput(const EstimatorManagerInput& emi) = default;
+  EstimatorManagerInput(EstimatorManagerInput&& emi)      = default;
+  EstimatorManagerInput& operator=(const EstimatorManagerInput& emi) = default;
+  EstimatorManagerInput& operator=(EstimatorManagerInput&& emi) = default;
+  // This constructor is for merging the global and local EstimatorManagerInput.
+  EstimatorManagerInput(const EstimatorManagerInput& global_emi, const EstimatorManagerInput& local_emi);
```

Comment:
This is not generic. It cannot handle three or more inputs to be merged. I'm thinking of just `EstimatorManagerInput::mergeInput(const EstimatorManagerInput& emi);` to add emi to *this.
On the use side,
```
EstimatorManagerInput merged_input(global_emi);
merged_input.mergeInput(local_emi);
```

Reply:
I figured out a way to write it generically and have it be construction time.

Reply:
Liked the new way.